docs/jinja.md

```diff
@@ -0,0 +1,71 @@
+# Generator framework based on Jinja template engine
+
```

Comment:
Maybe add a note ("Warning") as in "projects scaffolding" mentioning that one need to install another project...

Reply:
Agreed. Done.